### DISCUSSION TITLE
Older LTS

### DISCUSSION THREAD

**Discussion by arahmitz** [Victor-IX/Blender-Launcher-V2#22] [General]
Hey, 
Am I the only one missing older LTS? 2.93, 3.3 arent available, just the daily ones. Will it be fixed or am I missing something?
![image](https://github.com/Victor-IX/Blender-Launcher-V2/assets/34507800/10bc5031-ed8c-4ba3-a75b-eb1fd8174064)

**Comment by zeptofine** [comment]
As the release notes show, We currently can't scrape stable releases due to issues on Blender's side. Their infrastructure was not really designed for our kind of tools. @Victor-IX has gottten in contact with a few Blender employees and they are designing some *official* apis for tools like ours to use! So all we can do right now is to wait

**Reply by arahmitz** [reply]
Thank you for your reply, at least I know why is it happening and that it's not on my side.
